rfc/0002-subdocapi.md

```diff
@@ -179,11 +208,12 @@ Be aware that for lookup operations which fail partially in languages which can
 These errors are only received for `counter_in` operations:
 
 * NumberTooBig: Existing number value in document is too big (out of range between `INT64_MIN` and `INT64_MAX`
-* ZeroDelta: Got a zero value for a delta
+* BadDelta: This corresponds to the `PROTOCOL_BINARY_RESPONSE_DELTA_ERANGE`
```

Comment:
See http://review.couchbase.org/#/c/60052/ - I plan to fix the current delta handling.